samples/algorithms/Shor.qs

```diff
@@ -11,6 +11,7 @@ namespace Sample {
     open Microsoft.Quantum.Random;
     open Microsoft.Quantum.Math;
     open Microsoft.Quantum.Arithmetic;
+    open Microsoft.Quantum.Unstable.Arithmetic;
```

Comment:
I curious about the reason for using unstable functions/operations on our samples? Wouldn't it be preferable to use stable ones on our own samples code?

Reply:
This sample uses IncByLE function from the unstable namespace because we don't have anything stable for it yet. We don't want to include this function in the standard library, but we don't have any other way of distributing library code yet. Once we have a way to distribute libraries, we will remove this from here. Hence it is unstable.